src/coreclr/src/utilcode/longfilepathwrappers.cpp

```diff
@@ -11,14 +11,14 @@
 class LongFile
 {
 private:
-#ifndef TARGET_UNIX
+#ifndef HOST_UNIX
```

Comment:
This would better read as `HOST_WINDOWS` because of the code under it is very Windows-specific.